internal/serve/serve.go

```diff
@@ -0,0 +1,48 @@
+package serve
+
+import (
+	"fmt"
+	"net/http"
+
+	supporthttp "github.com/stellar/go/support/http"
+	supportlog "github.com/stellar/go/support/log"
+	"github.com/stellar/go/support/render/health"
+)
+
+type Configs struct {
+	Logger *supportlog.Entry
+	Port   int
+}
+
+type handlerDeps struct {
+	Logger *supportlog.Entry
+}
+
+func Serve(cfg Configs) {
+	deps := getHandlerDeps(cfg)
+
+	addr := fmt.Sprintf(":%d", cfg.Port)
+	supporthttp.Run(supporthttp.Config{
+		ListenAddr: addr,
+		Handler:    handler(deps),
+		OnStarting: func() {
```

Comment:
Can we add the `OnStopping` as well? So we can log and also handle gracefully other things in the future.

Reply:
https://github.com/stellar/wallet-backend/pull/2/commits/323a4cf3e9f945ebec42c5b58008dfe4a3759df8